stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -1304,6 +1308,13 @@ public class ChatClient internal constructor(
             warmUp = false
         }
 
+        @VisibleForTesting
```

Comment:
Sorry for that. Better to add TLS support to `MockWebServer` (see `MockWebServer.useHttps()` method below). However, in that case, client needs to trust this self-signed certificate.

Reply:
I think we should at least mark it with `@InternalStreamChatApi` annotation

Reply:
IMO it's ok to expose it as `@InternalStreamChatApi`. If someone strongly decided to suppress such warning and use it then it is his fault

Reply:
I guess there will be no warnings in java